sequence/arguments/tests/Test.js

```diff
@@ -1,44 +1,37 @@
 const {
-  runGqlOk,
-  expectData,
-  deployEndpoint,
+  deployAndRun,
+  KeyType,
 } = require("../../../tests/gqltest.js");
 
-const endpoint = process.env.STEPZEN_ENDPOINT;
-
 describe("sequence/arguments", function () {
+  const tests = [
+    { label: "customer(1)", 
+      fieldSelection: "{customer(id:1){name city weather{temp}}}", 
+      expected: {customer: { name: "John Doe", city: "Miami", weather: { temp: 100.0, }}},
+      keyType: KeyType.admin,
```

Comment:
probably a future improvement, but it would be better if there was a default for the key type, so I don't have to specify on each test.